src/application/menus.css

```diff
@@ -0,0 +1,172 @@
+
+
+.p-MenuBar {
+  padding-left: 5px;
+  background: var(--jp-layout-color1);
+  color: var(--jp-ui-font-color1);
+  font-size: var(--jp-ui-fontSize1);
+}
+
+
+.p-MenuBar-menu {
+  transform: translateY(calc(-1*var(--jp-borderWidth)));
+}
+
+
+.p-MenuBar-item {
+  padding: 0px 8px;
+  border-left: var(--jp-borderWidth) solid transparent;
+  border-right: var(--jp-borderWidth) solid transparent;
+  line-height: var(--jp-private-menubar-height);
+}
+
+
+.p-MenuBar-item.p-mod-active {
+  background: var(--jp-layout-color2);
+}
+
+
+.p-MenuBar.p-mod-active .p-MenuBar-item.p-mod-active {
+  z-index: 10001;
+  border-left: var(--jp-borderWidth) solid var(--jp-borderColor1);
+  border-right: var(--jp-borderWidth) solid var(--jp-borderColor1);
+  box-shadow: 0px 0px 6px rgba(0, 0, 0, 0.2);
+}
+
+
+.p-MenuBar-item.p-mod-disabled {
+  color: var(--jp-ui-font-color2);
+}
+
+
+.p-MenuBar-item.p-type-separator {
+  margin: 2px;
+  padding: 0;
+  border: none;
+  border-left: var(--jp-borderWidth) solid var(--jp-borderColor2);
+}
+
+
+.p-MenuBar-itemMnemonic {
+  text-decoration: underline;
+}
+
+
+#jp-top-panel {
+  border-bottom: var(--jp-borderWidth) solid var(--jp-borderColor1);
+  background: var(--jp-layout-color1);
+  display: flex;
+  min-height: var(--jp-private-menubar-height);
+}
+
+
+/*-----------------------------------------------------------------------------
+| Menu
+|----------------------------------------------------------------------------*/
+
+
+.p-Menu {
+  z-index: 10000;
+  padding: 4px 0px;
+  background: var(--jp-layout-color1);
+  color: var(--jp-ui-font-color1);
+  border: var(--jp-borderWidth) solid var(--jp-borderColor1);
+  font-size: var(--jp-ui-fontSize1);
+  box-shadow: 0px 1px 6px rgba(0, 0, 0, 0.2);
+}
+
+
+.p-Menu-item {
+  min-height: var(--jp-private-menu-item-height);
+  max-height: var(--jp-private-menu-item-height);
+  padding: 0px;
+  line-height: var(--jp-private-menu-item-height);
+}
+
+
+.p-Menu-item.p-mod-active {
+  background: var(--jp-layout-color2);
+}
+
+
+.p-Menu-item.p-mod-disabled {
+  color: var(--jp-ui-font-color2);
+}
+
+
+.p-Menu-itemIcon {
+  width: 21px;
+  padding: 0px 2px 0px 4px;
+
+}
+
+
+.p-Menu-itemLabel {
+  padding: 0px 32px 0px 2px;
+}
+
+
+.p-Menu-itemMnemonic {
+  text-decoration: underline;
+}
+
+
+.p-Menu-itemShortcut {
+  padding: 0px;
+}
+
+
+.p-Menu-itemSubmenuIcon {
+  width: 16px;
+  padding: 0px 4px 0px 0px;
+}
+
+
+.p-Menu-item.p-type-separator > div {
+  padding: 0;
+  height: 9px;
+}
+
+
+.p-Menu-item.p-type-separator > div::after {
+  content: '';
+  display: block;
+  position: relative;
+  top: 4px;
+  border-top: var(--jp-borderWidth) solid var(--jp-layout-color2);
+}
+
+.p-Menu-itemIcon::before,
+.p-Menu-itemSubmenuIcon::before {
+  font-family: FontAwesome;
+}
+
+
+.p-Menu-item.p-mod-toggled > .p-Menu-itemIcon::before {
+  content: '\f00c';
+}
+
+
+.p-Menu-item.p-type-submenu > .p-Menu-itemSubmenuIcon::before {
+  content: '\f0da';
```

Comment:
Does it make sense to have FontAwesome glyph unicode values in variables as well? If we do this in addition to making the font value: `FontAwesome` itself a variable, it would allow a theme author to use a completely different set of glyphs from a different library, perhaps.